### PR TITLE
#3398 temperature logs in generated data

### LINKED ISSUE #3398: Temperature logs in generated data to be a little random
## Is your feature request related to a problem? Please describe.

Think it would be good to have data generated which was less 'random'. This is to give better visual testing of data vis - i.e. the chart. Where it'd be easier to test that 'normal' cases where temperatures are usually OK with occasional breaches, and 'abnormal' cases where the temperature is just some stupid high number or stupid low number, looks OK.

Also would be good for a lot more data to be generated. Good to test the performance of the chart with a large amount of data (to ensure it is OK or if we need to add constraints.

## Describe the solution you'd like

Maybe something like

i.e.:

Sensor1: Temperatures ranging from 2-6 with big breach spikes - a little more 'realistic'
Sensor2: Temperatures realllly high all the time
Sensor3: Temperatures reallly low all the time
Sensor4: temperatures with no breaches in the normal range

## Implementation

TBD

## Describe alternatives you've considered

N/A

## Additional context

N/A

### PR DESCRIPTION
**Pull request by mark-prins**
Fixes #3398 

## Change summary

Changed the methods which generate the temperature logs.
- The five types now align to the sensors i.e. sensor 1 is always the 'kind of ok, but with some breaches' one; sensor 2 is hot etc.
- Added a 'no temperature logs' to the `FridgeDetailPage` ( there's an error otherwise )
- Sorted the sensors on the `FridgeDetailPage` by name
- Added another dummy sensor, so that there are now these sensors, with data:
1. Ok, but some breaches
2. All hot
3. All cold
4. no breaches
5. no data at all

### Sensor 1

Screen Shot 2021-02-09 at 3 22 39 PM

### Sensor 2

Screen Shot 2021-02-09 at 3 22 51 PM

### Sensor 3

Screen Shot 2021-02-09 at 3 23 01 PM

### Sensor 4

Screen Shot 2021-02-09 at 3 23 10 PM

### Sensor 5

Screen Shot 2021-02-09 at 3 23 28 PM

## Testing

Steps to reproduce or otherwise test the changes of this PR:

- [ ] Generate data. This should run without error
- [ ] Sensor 1 shows data which is mainly between 2-6 degrees, with some breaches both high and low
- [ ] Sensor2 shows all high breaches
- [ ] Sensor3 shows all low breaches
- [ ] Sensor4 shows no breaches
- [ ] Sensor5 has no temperature logs and tells you this, without showing an error

### Related areas to think about

If there are any general areas of the codebase your changes might have side affects on, mention them here

### CODE REVIEW

**Review by joshxg — APPROVED** [review]
Ah thanks a lot, it's awesome. Good to see different data and a different perspective! Should really get some real data soon I suppose 🤣 😝

**Comment by joshxg** [comment]
did get a prop type error on sensor 5 

image

File: src/pages/FridgeDetailPage.js [hunk 1]
```diff
@@ -71,6 +74,19 @@ const BreachCard = props => (
   />
 );
 
+const EmptyComponent = ({ sensorName }) => (
+  <FlexView justifyContent="center" alignItems="center" flex={1}>
+    <BreachManUnhappy size={BREACH_MAN_UNHAPPY_SIZE} />
+    <Text style={localStyles.emptyText}>
+      No temperatures logged for {sensorName} within the given date range
```
Comment (joshxg) [inline src/pages/FridgeDetailPage.js:81]: localize?

File: src/pages/SettingsPage.js [hunk 2]
```diff
@@ -151,31 +170,43 @@ const Settings = ({ toRealmExplorer, currentUserPasswordHash, requestStorageWrit
...
+      const fridge5 = createRecord(UIDatabase, 'Location', { description: 'Fridge 5', code: 'f5' });
 
       // create sensors
       createRecord(UIDatabase, 'Sensor', {
         name: 'sensor 1',
-        macAddress: 'sen1',
+        macAddress: '00:00:00:00:00:01',
         batteryLevel: 10,
         location: fridge1,
+        isActive: false,
```
Comment (joshxg) [inline src/pages/SettingsPage.js:181]: Oh - why are you setting this as false? I just generated the data on this PR and hadn't noticed - I noticed only when I went back to my branch which I am working on which is interacting with `isActive` heh

File: src/pages/FridgeDetailPage.js (hunk 1 above)
Comment (mark-prins) [reply src/pages/FridgeDetailPage.js:81]: _localise_ 😄  ( 🇬🇧  )

File: src/pages/SettingsPage.js (hunk 2 above)
Comment (mark-prins) [reply src/pages/SettingsPage.js:181]: yeah, sorry. I'll revert that. I was using `isActive` to stop the bluetooth actions for fake sensors.

**Comment by mark-prins** [comment]
> did get a prop type error on sensor 5
> 
> image

actually #3468 will sort that out ( probably ). will recheck after merging 😄

File: src/pages/FridgeDetailPage.js (hunk 1 above)
Comment (mark-prins) [reply src/pages/FridgeDetailPage.js:81]: b66c695d

**Comment by joshxg** [comment]
> actually #3468 will sort that out ( probably ). will recheck after merging 😄

Haha I got confused between branches, whoops!

**Pull request merged**